app/components/Auction.js

```diff
@@ -63,9 +63,11 @@ export default class Auction extends Component {
       bidIncrement,
       duration,
       startedAt,
-      isActive,
+      startBlock,
+      status,
       highestBid,
-      highestBidder
+      highestBidder,
+      cancelled,
```

Comment:
Don't include `cancelled`, just `status`

Reply:
Oh yeah put that for debugging